src/com/trainee/phani/Maxiumelement.java

```diff
@@ -0,0 +1,22 @@
+package com.trainee.phani;
+import java.util.Scanner;
+//program to find a maximum element in an array
+public class Maxiumelement {
+    public static void main(String[] args){
+        Scanner sc=new Scanner(System.in);
+        System.out.println("enter n value");
+        int n=sc.nextInt();
+        System.out.println("enter the elements");
+        int []arr=new int[n];
+        for(int i=0;i<n;i++){
+            arr[i]=sc.nextInt();
+        }
+        int max=arr[0];
+        for(int i=1;i<n;i++){ //comparing with elements
```

Comment:
Do not put comment in the code line. Put it on top
If we need to make any change to the code then we need do deal with comment as well.
Better leave it in new line

Reply:
Also formatting is missing entire for statement